articles/aks/windows-faq.md

```diff
@@ -189,6 +189,13 @@ Set-TimeZone -Id "Russian Standard Time"
 
 To see the current time zone of the running container or an available list of time zones, use [Get-TimeZone](/powershell/module/microsoft.powershell.management/get-timezone).
 
+## Can I maintain session affinity from client connections to pods with Windows Containers?
+Only by a workaround. The product group is currently working to add AKS support for Windows Server 2022 images which will support session affinity with Kube-Proxy. The workaround to allow session affinity by Client IP can be achieved by limiting your desired pod to run a single instance per node and configuring your Kubernetes service to direct traffic to the pod on the local node. To achieve this, the following configuration can be used:
```

Comment:
Can you change the language from “only by a workaround” to something like “whilst this will be supported in WS2022, the current way to enable session affinity…”

Reply:
Updated the language